test/util/uk/gov/tariffclassificationfrontend/utils/WiremockTestServer.scala

```diff
@@ -37,6 +37,7 @@ trait WiremockTestServer extends UnitSpec with BeforeAndAfterEach {
   }
 
   override protected def afterEach(): Unit = {
+    wireMockServer.resetAll()
```

Comment:
for safety I would add `super.afterEach()` as first instruction here

Reply:
same for the `beforeEach` above

Reply:
Done